### ISSUE TITLE
Release Photos 1.5.0

### ISSUE THREAD

**Issue by offbyone** [pelican-plugins/photos#89]
Although I'm able to work with this by installing from git (see #70 for my issues) I'd love to get back to a pinned version at some point. Could you cut a 1.5.0 release please?

**Comment by justinmayer** [comment]
Hi @phibos! 👋  @offbyone and I have been making some improvements to the Photos plugin in the context of the impromptu [Pelican Sprint](https://github.com/getpelican/pelican/discussions/3212) I organized. Given the improvements that have been made since the last package was published a year ago, I would like to release Photos 1.5.0 today. Is that okay with you?

There is a pull request in the queue that I marked for your review. If you have the time to take a look, perhaps we can get it merged today and include it in the release. Otherwise it can wait until a future 1.6.0 version.

If there are no objections by, say, 20:00 today (UTC+1), I will plan on releasing the new 1.5.0 version. ✨ 

Feel free to join us in the chat today! https://web.libera.chat/?#pelican
